### DISCUSSION TITLE
App/Playtime: Add Linux support

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#977] [Ideas]
This will probably happen in 2 stages:

1. Make the app available as standalone app (not embedded into REAPER). This is not too much effort and would already make it usable, but with a few annoying UX issues. 
2. Embed the app into REAPER (as done on Windows and macOS).

**Comment by mikobuntu** [comment]
Just checking in to see if there has been any progress on this ? Thanks :)

**Reply by helgoboss** [reply]
No, busy with fixing bugs / working towards the final release (my first priority right now).

**Reply by mikobuntu** [reply]
OK thanks for the update, much appreciated :)

**Comment by helgoboss** [comment]
Guys, I'm starting to work on Linux support stage 1. You can follow the progress here: https://github.com/helgoboss/helgobox/issues/1391

**Comment by helgoboss** [comment]
[Linux support stage 1 is here!](https://github.com/helgoboss/helgobox/discussions/1414)

**Comment by helgoboss** [comment]
Experimental stage 2 support is here: https://github.com/helgoboss/helgobox/issues/1397#issuecomment-5700566618
